patches/react-native-web+0.19.9+004+fixLastSpacer.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
-index faeb323..68d740a 100644
+index 7f6c880..b05da08 100644
```

Comment:
These are just sha1 hashes representing the versions of the file before and after the changes. This is because I had this patch applied in this PR before it was moved to a separate PR. 
I've rechecked the changes, and they have been applied properly.

Reply:
Right, but since there are no changes in this file in this PR, the hashes should not change, right?

Reply:
As this change was made twice in different PRs, they have different IDs, but you're right, I can remove a new one, and it would work